### ISSUE TITLE
Deprecated constants

### ISSUE THREAD

**Issue by adrien-parasote** [maheus/fumis_integration#16]
Hello guys,

First of all thanks for the integration ^^ everything is working fine.
However, the following warnings/errors were raised in HA logs so I put them here for reporting and hopefully you can take them into account for the next component version.

**Warnings:** 
_deprecated constant which will be removed in HA Core 2025.1_
- DEVICE_CLASS_TEMPERATURE : use SensorDeviceClass.TEMPERATURE
- DEVICE_CLASS_POWER : use SensorDeviceClass.POWER
- DEVICE_CLASS_CURRENT : use SensorDeviceClass.CURRENT
- POWER_KILO_WATT : use UnitOfPower.KILO_WATT
- TEMP_CELSIUS : use UnitOfTemperature.CELSIUS
- SUPPORT_PRESET_MODE : use ClimateEntityFeature.PRESET_MODE
- SUPPORT_TARGET_TEMPERATURE : use ClimateEntityFeature.TARGET_TEMPERATURE
- CURRENT_HVAC_OFF : use HVACAction.OFF instead
- CURRENT_HVAC_HEAT : use HVACAction.HEATING instead
- CURRENT_HVAC_COOL : use HVACAction.COOLING instead

_Async call function_
Detected that custom integration 'fumis' calls async_forward_entry_setup for integration which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/fumis/__init__.py, line 97: hass.async_create_task(

**Comment by maheus** [comment]
Hello, 

Yes i know, thanks for this issue. I will take care of it  as soon as possible.

**Comment by adrien-parasote** [comment]
No worry we still have time until 2025.1 ^^
I’ve python background but unfortunatly no time yet to setup env, perform dev, test then PR. However if i can help just tell me.

**Comment by fudu** [comment]
Hi, what about this?
2025.1 is out
Will integration stop to work if ha upgrade?
Thanks

**Comment by maheus** [comment]
Yes i'm sorry, i fix it this weekLe 5 janv. 2025 19:59, fudu ***@***.***> a écrit :
Hi, what about this?
2025.1 is out
Will integration stop to work if ha upgrade?
Thanks

**Comment by gvandeveen** [comment]
After the 2025.1 update the integration doesn't work anymore.

Glad to see you will fix this soon @maheus 👍

**Comment by maheus** [comment]
@gvandeveen @fudu @adrien-parasote  can you test the 3.0.1-beta.1 release please.

**Comment by adrien-parasote** [comment]
Yes i just need to rollback z2mqtt < 2 before.
Keep you in touch.

**Comment by gvandeveen** [comment]
I have tested the 3.0.1-beta.1 release and the stove is back up again!

Thanks for the quick fix 👍 

Took me some time to rember to fill out the mac address as 00000000 instead of 00:00:00:00

**Comment by adrien-parasote** [comment]
Same here. I just deleted old sensor and replace *_2 then everything is ok.
Thanks

**Comment by maheus** [comment]
> I have tested the 3.0.1-beta.1 release and the stove is back up again!
> 
> Thanks for the quick fix 👍
> 
> Took me some time to rember to fill out the mac address as 00000000 instead of 00:00:00:00

yes i going to change the doc

**Comment by fudu** [comment]
I installe 3.0.1, it seems all ok
Thank you
